subsys/net/lib/coap/coap.c

```diff
@@ -763,6 +763,16 @@ static bool uri_path_eq(const struct coap_packet *cpkt,
 			continue;
```

Comment:
The commit message and subject is wrong in the commit that changes coap.c, could you fix that.

Reply:
Sorry, mistake during rebase